src/blueapi/worker/reworker.py

```diff
@@ -136,6 +136,21 @@ def submit_task(self, task: Task) -> str:
         self._tasks[task_id] = trackable_task
         return task_id
 
+    def get_tasks_by_status(self, status: str) -> list[TrackableTask[Task]]:
```

Comment:
Status should be an enum

Reply:
will do hectic refactoring tomorrow

Reply:
update: changing the argus to be a required instance of the enum , without the big refactor